### PR TITLE
Fix/aut 2772/text reader updater script fix

### PR DESCRIPTION
**Pull request by pnal**

### CODE REVIEW

File: model/LegacyPciHelper/ImageToPropertiesHelper.php [hunk 1]
```diff
@@ -24,14 +24,15 @@
...
+use oat\taoMediaManager\model\fileManagement\FlySystemManagement;
 use oat\taoMediaManager\model\MediaSource;
 
 class ImageToPropertiesHelper
 {
     /** @var MediaSource */
     private $mediaSource;
 
-    /** @var FileManagement */
+    /** @var FlySystemManagement */
```
Comment (andreluizmachado) [inline model/LegacyPciHelper/ImageToPropertiesHelper.php:35]: Why did you change it?

**Comment by andreluizmachado** [comment]
Hey, can you also check what happened with the CI?

**Review by bartlomiejmarszal — APPROVED** [review]

**Review by andreluizmachado — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by bugalot — APPROVED** [review]
I couldn't run it, but it looks good!
